src/yum-plugins/tracer_upload.py

```diff
@@ -0,0 +1,66 @@
+import sys
+import json
+import httplib
+import time
+
+try:
+  from tracer.query import Query
+except ImportError:
+  sys.exit('Error Importing tracer! Is tracer installed?')
+
+from yum.plugins import PluginYumExit, TYPE_CORE, TYPE_INTERACTIVE
+from rhsm.config import RhsmConfigParser, initConfig
+
+sys.path.append('/usr/share/rhsm')
+from subscription_manager.identity import ConsumerIdentity
+
+requires_api_version = '2.3'
+plugin_type = (TYPE_CORE, TYPE_INTERACTIVE)
+
+def query_apps(conduit):
+    """Returns all apps that need restarting """
+    query = Query()
+    if conduit:
+        pkgs = conduit.getTsInfo().getMembers()
+        packages = []
+        for pkg in pkgs:
+            pkg.modified = time.time()
+            packages.append(pkg)
```

Comment:
Of course after merging, i think i found an issue :)

It looks like this logic is only causing packages that were updated to be submitted and ignoring anything else.  This causes for example:

yum update mongodb
yum update tuned

to first report mongodb, then report *just* tracer, so the UI ends up showing only tracer.  When running katello-tracer-upload it properly reports both.  Any reason to just not report all services during a yum action?